src/couch/priv/couch_js/60/main.cpp

```
@@ -412,8 +412,10 @@ main(int argc, const char* argv[])
 
     couch_args* args = couch_parse_args(argc, argv);
 
-    JS_Init();
-    cx = JS_NewContext(args->stack_size, 8L * 1024L);
+    if(!JS_Init())
+        return 1;
```

I added this to be sure the initialization didn't fail silently.